sb.web.app/src/App.vue

```diff
@@ -61,7 +61,7 @@ export default {
       } 
     })
 
-    setStoreData()
+    // setStoreData()
```

Comment:
seems like a pretty signifcant change...

also leaving code commented can seem like a good idea at the time but it almost always degrades the quality of the code base.. definitely prefer removing entirely